### PR TITLE
Feature/add-circleci

### PR DESCRIPTION
**Pull request by ayslanmarcelino**

### CODE REVIEW

**Review by andersonfernandes — COMMENTED** [review]
Review done with comments

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          POSTGRES_USER: postgres
+          POSTGRES_DB: finances_test
+          POSTGRES_PASSWORD: ""
+
+    steps:
+      - checkout_and_bundle_install
+
+      - run:
+          name: Update binaries
+          command: sudo apt-get update && sudo apt-get install -y ghostscript xvfb libfontconfig wkhtmltopdf
```
[the remark this answers is not in the document]
Comment (ayslanmarcelino) [reply .circleci/config.yml:56]: No. I fixed.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    docker:
+      - image: circleci/ruby:2.6.5-node-browsers
+        environment:
+          BUNDLE_JOBS: 2
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: vendor/bundle
+          RAILS_ENV: test
+          DATABASE_URL: postgresql://postgres:@localhost/finances_test
+      - image: circleci/postgres:9.5-alpine
+        environment:
```
[the remark this answers is not in the document]
Comment (ayslanmarcelino) [reply .circleci/config.yml:46]: done

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  rspec:
+    parallelism: 2
+    docker:
+      - image: circleci/ruby:2.6.5-node-browsers
+        environment:
+          BUNDLE_JOBS: 2
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: vendor/bundle
+          RAILS_ENV: test
+          DATABASE_URL: postgresql://postgres:@localhost/finances_test
```
Comment (andersonfernandes) [inline .circleci/config.yml:44]: Add here the secret key

Comment (ayslanmarcelino) [reply .circleci/config.yml:44]: done

**Review by andersonfernandes — APPROVED** [review]

**Pull request merged**